### PR TITLE
Add test for virsh connection early in setup

### PR DESCRIPTION
**Pull request by hamzy**

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml [hunk 1]
```diff
@@ -482,6 +487,10 @@ objects:
           exit 1
         fi
 
+        echo "JOB_NAME_HASH=${JOB_NAME_HASH}"
+        echo 'network:'
+        echo '  if: "br'$(printf ${LEASED_RESOURCE} | tail -c 3)'-'${JOB_NAME_HASH}'"'
+
```
Comment (jaypoulz) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml:493]: Shouldn't this info be printed out with the install-config.yaml?
If the install-config.yaml isn't already being printed out, can we do that instead?

Comment (hamzy) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml]: As discussed, I cated out the entire file...

**Review by jaypoulz — APPROVED** [review]
Looks good! :)
Well need an ack from one of the template owners though.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**